tests/unit/Imports/ProductImportTest.php

```diff
@@ -155,6 +157,209 @@ function (MockInterface $module) {
         $this->assertCount(1, $variableProduct->get_upsell_ids(), '1 upsell IDs should be set');
     }
 
+    public function dataProviderTestGetStockProperties()
+    {
+        $tests = [];
+
+        $tests['simple or configurable assign product in stock, limited, has 0 orderable stock'] = [
```

Comment:
```suggestion
        $tests['simple  product in stock, limited, has 0 orderable stock'] = [
```

can you also change other test names ?

Reply:
Corrected test names